backend/migration/src/m20250129_162814_add_team_invitation_field_to_user.rs

```diff
@@ -0,0 +1,39 @@
+use sea_orm_migration::prelude::*;
+
+#[derive(DeriveMigrationName)]
+pub struct Migration;
+
+#[async_trait::async_trait]
+impl MigrationTrait for Migration {
+    async fn up(&self, manager: &SchemaManager) -> Result<(), DbErr> {
+        manager
+            .alter_table(
+                Table::alter()
+                    .table(Users::Table)
+                    .add_column(
+                        ColumnDef::new(Users::TeamInvitations)
+                            .array(ColumnType::String(StringLen::None))
```

Comment:
Noooo, we use postgress. Please create new table, define proper relations, unique keys, constrains etc